src/pages/generate/pages/result/curationSheet/CurationSheet.tsx

```
@@ -271,9 +252,12 @@ export const CurationSheet = ({ groupId = null }: CurationSheetProps) => {
     if (activeImageId === null) {
       return renderStatus(
         '가구 추천을 보려면 생성된 이미지를 먼저 선택해 주세요',
-        '결과 이미지에서 핫스팟을 선택하면 추천이 표시돼요'
+        '상단 가구 필터에서 원하는 가구를 선택해 주세요'
       );
     }
+    if (!hasDetectionCodes) {
+      return renderStatus('가구를 분석 중이에요', '잠시만 기다려 주세요');
```

ESLint

[error] Replace `'가구를·분석·중이에요',·'잠시만·기다려·주세요'` with `"가구를·분석·중이에요",·"잠시만·기다려·주세요"`

(prettier/prettier)